sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StandardMetricsExtractionProcessor.cs

```diff
@@ -79,6 +86,7 @@ private void ReportRequestDurationMetric(Activity activity)
             tags.Add(new KeyValuePair<string, object?>(StandardMetricConstants.CloudRoleInstanceKey, StandardMetricResource?.RoleInstance));
             tags.Add(new KeyValuePair<string, object?>(StandardMetricConstants.CloudRoleNameKey, StandardMetricResource?.RoleName));
             tags.Add(new KeyValuePair<string, object?>(StandardMetricConstants.RequestSuccessKey, RequestData.IsSuccess(activity, statusCodeAttributeValue, OperationType.Http)));
+            tags.Add(new KeyValuePair<string, object?>(StandardMetricConstants.IsSyntheticKey, IsSyntheticRequest(userAgent) ? "True" : "False"));
```

Comment:
not part of this PR, but just noticed that most of the other tags for the metric is fixed. So instead of passing them with each Record call, they should be modelled as resource/meter tags, and let exporter extract and puts them in proper place. This is significant perf overhead otherwise.

Take a look at the benchmarks to see how the perf is impacted with each additional dimension!
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/Benchmarks/Metrics/HistogramBenchmarks.cs#L22-L25

(Not a blocker for *this* PR)